app/src/main/res/values/strings.xml

```diff
@@ -279,17 +279,17 @@
   <string name="nearby_location_has_not_changed">Location has not changed.</string>
   <string name="nearby_location_not_available">Location not available.</string>
   <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
-  <string name="get_directions">GET DIRECTIONS</string>
+  <string name="get_directions">Get Directions</string>
   <string name="read_article">READ ARTICLE</string>
 
   <string formatted="false" name="notifications_welcome">Welcome to Wikimedia Commons, %1$s! We\'re glad you\'re here.</string>
   <string name="notifications_talk_page_message">%1$s left a message on your talk page</string>
   <string name="notifications_thank_you_edit">Thank you for making an edit</string>
   <string name="notifications_mention">%1$s mentioned you on %2$s.</string>
   <string name="toggle_view_button">Toggle view</string>
-  <string name="nearby_directions">DIRECTIONS</string>
-  <string name="nearby_wikidata">WIKIDATA</string>
-  <string name="nearby_wikipedia">WIKIPEDIA</string>
+  <string name="nearby_directions">Directions</string>
+  <string name="nearby_wikidata">Wikidata</string>
+  <string name="nearby_wikipedia">Wikipedia</string>
   <string name="nearby_commons">COMMONS</string>
```

Comment:
Hi, it seems like this one is forgotten

Reply:
@neslihanturan  the change you requested is done